h2o-docs/src/product/data-science/naive-bayes.rst

```diff
@@ -94,6 +94,74 @@ By default, the following output displays:
 -  A Priori response probabilities
 -  P-conditionals
 
+Examples
+~~~~~~~~
+
+Below is a simple example showing how to build a Naïve Bayes Classifier model.
+
+.. tabs::
+   .. code-tab:: r R
```

Comment:
Another example that doesn't use any NB parameters. Can you maybe switch to the prostate dataset and add laplace=0 like here?
- https://github.com/h2oai/h2o-3/blob/master/h2o-py/tests/testdir_algos/naivebayes/pyunit_prostateNB.py
- https://github.com/h2oai/h2o-3/blob/master/h2o-r/tests/testdir_algos/naivebayes/runit_naivebayes_prostate.R